### PR TITLE
OSDOCS-8763: Documented the 4.14.3 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-8763](https://issues.redhat.com/browse/OSDOCS-8763)

Version(s):
4.14

Link to docs preview:
[4.14 Release notes](https://68188--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-3)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3100,6 +3100,34 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-3"]
+=== RHSA-2023:7315 - {product-title} 4.14.3 bug fix and security update
+
+Issued: 2023-11-21
+
+{product-title} release 4.14.3, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7315[RHSA-2023:7315] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7321[RHBA-2023:7321] advisory.
```
Comment (agantony) [inline release_notes/ocp-4-14-release-notes.adoc:3108]:
```suggestion
{product-title} release 4.14.3, which includes security updates, is now available. The list of bug fixes included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7315[RHSA-2023:7315] advisory. The RPM packages included in the update can be found at link:https://access.redhat.com/errata/RHBA-2023:7321[RHBA-2023:7321].
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3100,6 +3100,34 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.14.3 --pullspecs
+----
+
+[id="ocp-4-14-3-bug-fixes"]
+==== Bug fixes
+
+* Previously, `LoadBalancer` services were not created for a deployment when a node contained additional ports that each had the `enable_port_security` parameter set to `false`. Now, `LoadBalancer` services are created for a deployment that contains additional ports with this setting. (link:https://issues.redhat.com/browse/OCPBUGS-22974[*OCPBUGS-22974*])
```
Comment (agantony) [inline release_notes/ocp-4-14-release-notes.adoc:3122]:
```suggestion
* Previously, `LoadBalancer` services were not created for a deployment if a node contained additional ports with the `enable_port_security` parameter set to `false`. With this release, `LoadBalancer` services are created for a deployment that contains additional ports with this setting. (link:https://issues.redhat.com/browse/OCPBUGS-22974[*OCPBUGS-22974*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -3100,6 +3100,34 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, a `ClusterAutoscaler` resource would go into a `CrashBackoff` loop for nodes configured with the Container Storage Interface (CSI) implementation. Now because of dependency updates, a `ClusterAutoscaler` resource no longer goes into this loop for nodes configured in this way. (link:https://issues.redhat.com/browse/OCPBUGS-23270[*OCPBUGS-23270*])
```
Comment (agantony) [inline release_notes/ocp-4-14-release-notes.adoc:3124]:
```suggestion
* Previously, a `ClusterAutoscaler` resource would go into a `CrashBackoff` loop for nodes configured with the Container Storage Interface (CSI) implementation. With this release, due to dependency updates, a `ClusterAutoscaler` resource now no longer goes into this loop for nodes configured in this way. (link:https://issues.redhat.com/browse/OCPBUGS-23270[*OCPBUGS-23270*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -3100,6 +3100,34 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[id="ocp-4-14-3-updating"]
+==== Updating
+
+To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Comment (agantony) [inline release_notes/ocp-4-14-release-notes.adoc:3129]:
```suggestion
To update an existing {product-title} 4.14 cluster to the latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

**Review by agantony — COMMENTED** [review]
Good work! Added a few suggestions; overall lgtm!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:3108]: This is templated text as this stage that has went through a few rounds of approvals, so I'll leave the format as it. Thanks!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 4 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:3129]: Template text. I will leave the original format as it. Thanks.

**Comment by dfitzmau** [comment]
Hi @agantony . I updated the text. but the template changes are out of scope. Would you be OK to change the label status?

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**